### PR TITLE
There are lint errors in FinancialAidComponent that need to be fixed #109

### LINKED ISSUE #109: There are lint errors in FinancialAidComponent that need to be fixed
src/app/financial-aid/financial-aid.component.spec.ts:9:14
- [ ] WARNING: 9:14    deprecation  async is deprecated: use `waitForAsync()`, (expected removal in v12)

src/app/financial-aid/financial-aid.component.ts:12:3
- [ ] ERROR: 12:3    typedef      expected call-signature: 'ngOnInit' to have a typedef

### PR DESCRIPTION
**Pull request by sjpcp5**
change async to waitForAsync
add : void after ngOnInit()

☑︎ Verified app loads and loads application
☑︎ Ran lint and verified lint errors are addressed
☑︎ Ran tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1001.6",
+    "@angular-devkit/build-angular": "^0.1002.0",
```
Comment (alisaduncan) [inline package.json:42]: Would it be possible to revert this change and the package-lock change please? Thanks!

**Review by alisaduncan — COMMENTED** [review]
One change requested as noted below. You're almost there!

File: package.json (hunk 1 above)
Comment (sjpcp5) [reply package.json]: Yes I reverted the commit and made changes to just financial-aid.component.ts

**Review by alisaduncan — APPROVED** [review]
fixes #109
